store/models.py

```diff
@@ -17,16 +19,27 @@ def __str__(self):
         return f'{self.title} by {self.author}'
 
 
+class UserRatings(models.Model):
+    book = models.ForeignKey(Book, on_delete=models.CASCADE)
+    user = models.ForeignKey(
+        User, related_name='borrowerRating', null=True, blank=True, on_delete=models.SET_NULL)
+    rating = models.PositiveIntegerField(
+        validators=[MinValueValidator(0), MaxValueValidator(10)])
```

Comment:
For every UserRating, the user can't be null, and better use `models.CASCADE` here.
Nice use of validators!